script/input/5/integration.json

```diff
@@ -0,0 +1,19 @@
+{
+    "domains": {
+        "root": {
+            "chainlog": "0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F"
+        },
+        "optimism": {
+            "rpc": "OPT_RPC_URL",
+            "block": "OPT_BLOCK",
+            "l1Messenger": "<TODO>",
+            "l2Messenger": "<TODO>"
+        },
+        "arbitrum": {
+            "rpc": "ARB_RPC_URL",
+            "block": "ARB_BLOCK",
+            "inbox": "<TODO>",
+            "arbSys": "<TODO>"
```

Comment:
@hexonaut or @gbalabasquer do these exist on opt/arb goerli?

Reply:
Found this for Arbitrum: https://developer.arbitrum.io/useful-addresses (which I'll induce to say the inbox is https://goerli.etherscan.io/address/0x6BEbC4925716945D46F0Ec336D5C2564F419682C).
I guess arbSys is also address 0x as that should be on the L2 so they can take these special addresses.

Reply:
Yes both exist for Goerli and @gbalabasquer is right that the L2 addresses are the same on both.

Reply:
@hexonaut do you have handy the optimism L1 goerli one? I can't easily find it...

Reply:
I don't. It should be findable with a bit of hunting.

Reply:
Easy way to find this is look at the DAI bridge deploys. You can find them in the chainlogs.

`l1Messenger`: 0x5086d1eEF304eb5284A0f6720f79403b4e9bE294
`l2Messenger`: 0x4200000000000000000000000000000000000007
`inbox`: 0x6BEbC4925716945D46F0Ec336D5C2564F419682C
`arbSys`: 0x0000000000000000000000000000000000000064

Reply:
Ah right! thanks